apps/apollo-stories/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@axa-fr/apollo-stories",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "storybook dev -p 6410",
+    "build": "storybook build",
+    "check-types": "tsc --noEmit",
+    "eslint": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
+    "eslint:fix": "eslint src --ext js,jsx,ts,tsx --fix",
+    "prettier": "prettier \"src/**/*.!(js|jsx|ts|tsx|svg)\" --check",
+    "prettier:fix": "prettier \"src/**/*.!(js|jsx|ts|tsx|svg)\" --write",
+    "chromatic": "npx chromatic"
+  },
+  "dependencies": {
+    "@axa-fr/design-system-apollo-react": "*",
+    "@axa-fr/design-system-apollo-css": "*",
+    "@material-symbols/svg-400": "^0.28.1"
+  },
+  "devDependencies": {
+    "@chromatic-com/storybook": "^3.2.4",
+    "@storybook/addon-essentials": "^8.5.5",
+    "@storybook/addon-interactions": "^8.5.5",
+    "@storybook/addon-onboarding": "^8.5.5",
+    "@storybook/blocks": "^8.5.5",
+    "@storybook/react": "^8.5.5",
+    "@storybook/react-vite": "^8.5.5",
+    "@storybook/test": "^8.5.5",
+    "@vitejs/plugin-react": "^4.3.4",
+    "globals": "^15.14.0",
+    "storybook": "^8.5.5",
+    "typescript": "~5.7.2",
+    "vite": "^6.1.0"
+  }
```

Attention on est en storiebook 8.6.3 
```suggestion
  "devDependencies": {
    "@chromatic-com/storybook": "^3.2.5",
    "@storybook/addon-a11y": "^8.6.3",
    "@storybook/addon-essentials": "^8.6.3",
    "@storybook/addon-interactions": "^8.6.3",
    "@storybook/addon-onboarding": "^8.6.3",
    "@storybook/blocks": "^8.6.3",
    "@storybook/react": "^8.6.3",
    "@storybook/react-vite": "^8.6.3",
    "@storybook/test": "^8.6.3",
    "@vitejs/plugin-react": "^4.3.4",
    "chromatic": "^11.26.1",
    "globals": "^15.14.0",
    "storybook": "^8.6.3",
    "typescript": "~5.7.2",
    "vite": "^6.1.0"
  }
```